### DISCUSSION TITLE
Results different than with Google

### DISCUSSION THREAD

**Discussion by azertylr** [searxng/searxng#3212] [General]
hello,

I have an issue similar to this one : https://github.com/searx/searx/issues/1236
My searxNg server is located in DE and I would like it to behave like if I were in France.

When I compare results between Google and SearxNG (with only google enable) results are different.
I've tried for language : "all" "auto" "fr "fr-FR" "fr-XX" "XX-FR" but it is the same.

Whoogle is giving me the same result that Google with WHOOGLE_CONFIG_COUNTRY=fr

Do you have suggestion regarding SearxNg configuration please?

Best

**Comment by unixfox** [comment]
Related https://github.com/searxng/searxng/issues/2438
